composer.json

```
@@ -14,14 +14,14 @@
     "ext-gmp": "*",
     "ext-json": "*",
     "codemix/yii2-streamlog": "^1.3",
-    "simplesamlphp/simplesamlphp": ">=2.2.2 <2.3.0",
+    "simplesamlphp/simplesamlphp": ">=2.2.2 <2.5.0",
```

Do you want to force it to be a 2.4 version (rather than still allowed 2.2 and 2.3)? I don't know if any of the changes we're making to be compatible with 2.4 are possibly not backwards compatible, but it seems like we wouldn't want to downgrade to a pre-2.4 version at this point.
```suggestion
    "simplesamlphp/simplesamlphp": ">=2.4.0 <2.5.0",
```